pycheribuild/projects/cross/freertos.py

```diff
@@ -82,26 +89,48 @@ def __init__(self, config: CheriConfig):
         if self.target_info.target.is_cheri_purecap():
             # CHERI-RISC-V sophisticated Demo with more advanced device drivers
             # and currently only runs on FPGA-GFE, purecap
-            self.freertos_demos.append("RISC-V_Galois_P1")
-            self.demo_apps["RISC-V_Galois_P1"] = ["main_blinky", "main_netboot"]
+            self.supported_freertos_demos.append("RISC-V_Galois_P1")
+            self.supported_demo_apps["RISC-V_Galois_P1"] = ["main_blinky", "main_netboot"]
 
             self.make_args.set(EXTENSION="cheri")
 
+    @classmethod
+    def setup_config_options(cls, **kwargs):
+        super().setup_config_options(add_common_cross_options=False, **kwargs)
+
+        cls.demo = cls.add_config_option(
+            "demo", metavar="DEMO", show_help=True,
+            default=cls.default_demo,
+            help="The FreeRTOS Demo build (default: RISC-V-Generic)")  # type: str
```

Comment:
```suggestion
            help="The FreeRTOS Demo build")  # type: str
```
This currently prints
```
  --freertos/demo DEMO  The FreeRTOS Demo build (default: RISC-V-Generic)
                        (default: 'RISC-V-Generic')
  --freertos/prog PROG  The FreeRTOS program to build (default: main_blinky)
                        (default: 'main_blinky')
```